script/openqa-bootstrap

```diff
@@ -32,6 +32,9 @@ pkgs=(openQA-single-instance qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils
 if [ "$(uname -m)" = "aarch64" ]; then
     pkgs+=(qemu-uefi-aarch64)
 fi
+if [[ $setup_web_proxy == "nginx" ]] ; then
+    pkgs+=(nginx)
+fi
```

Comment:
I suggest to solve that as part of the package already. Everything we can do in the package we don't need to do in bootstrap, i.e.:
```suggestion
- pkgs=(openQA-single-instance qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils os-autoinst-distri-opensuse-deps)
if [[ $setup_web_proxy == "nginx" ]] ; then
    pkgs=(openQA-single-instance-nginx)
else
    pkgs=(openQA-single-instance)
fi
+ pkgs+=(qemu-arm qemu-ppc qemu-x86 qemu-tools sudo iputils os-autoinst-distri-opensuse-deps)
```